src/OidcClientSettings.ts

```diff
@@ -168,6 +175,7 @@ export class OidcClientSettingsStore {
         mergeClaims = false,
         // other behavior
         stateStore,
+        refreshTokenCredentials,
```

Comment:
refreshTokenCredentials = "same-origin"

See comment above, why...

Reply:
yes, will do. though about this too :)